translate/convert/factory.py

```diff
@@ -64,9 +61,6 @@ def __str__(self):
             msg += ' with template %s' % (self.templ_ext)
         return msg
 
-    def __unicode__(self):
-        return unicode(str(self))
```

Comment:
Just wondering if these should be adjusted to use `@six.python_2_unicode_compatible` as done below.